### PR TITLE
Refactoring/make dev-install

### PR DESCRIPTION
**Pull request by z4y4ts**
Replace `requirements.txt` with `make install`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,8 +26,8 @@ quality-py:
 quality-js:
 	eslint video_xblock/static/js/
 
-deps:
-	pip install --process-dependency-links -r requirements.txt
+install:
+	pip install --process-dependency-links -e .
```
Comment (polesye) [inline Makefile]: is it better than `python setup.py install`?

Comment (z4y4ts) [inline Makefile]: @polesye yes, this way we can leverage Travis cache and shorten build time.

Comment (z4y4ts) [inline Makefile]: I can add a comment here to explain a decision

Comment (polesye) [inline Makefile]: yes, it'd be great

**Pull request merged**